src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -39,6 +41,7 @@ export class ReadingsGraphComponent implements OnDestroy {
   public polyGraphData: any;
   public timeDropDownOpened = false;
   public isModalOpened = false;
+  public showResetButton = false;
```

Comment:
though we don;t have any other reset button, but better to call it `showResetZoomButton`

Reply:
The zoom looks good, but a couple things we also need to do:
* It will need to be able to zoom out as far as is possible, and I'm not sure how this design will do that. I.e. if the default shows 30 minutes of data, this video shows zooming in to say 1 minute of data and that's great. But it will also need to be able to zoom back out to say 1 hour or 8 hours of data.
* Equally important as zooming is panning. In fact, this is more important, as currently, there's no way to see past data in the product. There needs to be a way to graphically drag left/right (or perhaps other action) to move the graph to the past or the future.